### ISSUE TITLE
Should BatchNorm layers exist in the discriminator according to the paper?

### ISSUE THREAD

**Issue by endeavormoquan** [Lornatang/WassersteinGAN_GP-PyTorch#2]
According to the Sec. 4 and Sec. 5.2 in the paper, should we replace the batchnorm to the layernorm?
